pkg/render/manager.go

```diff
@@ -94,15 +95,20 @@ func Manager(
 	clusterDomain string,
 	esLicenseType ElasticsearchLicenseType,
 ) (Component, error) {
-	tlsSecrets := []*corev1.Secret{tlsKeyPair}
-	tlsSecrets = append(tlsSecrets, secret.CopyToNamespace(ManagerNamespace, tlsKeyPair)...)
+	var tlsSecrets []*corev1.Secret
 	tlsAnnotations := make(map[string]string)
+	if installation.CertificateManagement == nil {
+		tlsSecrets = append(tlsSecrets, tlsKeyPair)
```

Comment:
I do not like the fact that we are setting this secret to nil in the controller and checking on different condition here.